include/conversation.php

```diff
@@ -579,12 +579,11 @@ function conversation(App $a, array $items, $mode, $update, $preview = false, $o
 	$items = $cb['items'];
 
 	$conv_responses = [
-		'like' => ['title' => DI::l10n()->t('Likes','title')],
-		'dislike' => ['title' => DI::l10n()->t('Dislikes','title')],
-		'attendyes' => ['title' => DI::l10n()->t('Attending','title')],
-		'attendno' => ['title' => DI::l10n()->t('Not attending','title')],
-		'attendmaybe' => ['title' => DI::l10n()->t('Might attend','title')],
-		'announce' => ['title' => DI::l10n()->t('Reshares','title')]
```

Comment:
@MrPetovan Looks like you removed the 'announce' response. Is this done on purpose or by accident?

Reply:
Probably by accident, do reshare counts don't show anymore?

Reply:
Exactly! I just wanted to do the bugfix PR after I found the solution by re-adding a line for 'announce'. :-)

Reply:
You're getting very good at this!